vividus-plugin-web-app-to-rest-api/src/main/resources/vividus-plugin/spring.xml

```diff
@@ -88,6 +96,5 @@
     <util:list id="stepBeanNames-WebAppToRestApi" value-type="java.lang.String">
         <idref bean="httpRequestSteps" />
         <idref bean="resourceCheckSteps" />
-        <idref bean="redirectValidationSteps" />
```

Comment:
why is it removed? why is `sslLabsSteps` not added?

Reply:
> why is it removed? why is `sslLabsSteps` not added?

Sorry, just rebase issue